src/store/cart-store.js

```diff
@@ -7,6 +7,11 @@ class CartStore extends Store {
     getItems() {
         return items;
     }
+
+    clearItems() {
+        items = [];
```

Comment:
Store methods should return state, not mutate it.

Please try to use actions to mutate state. For example, a "CLEAR_ITEMS" action.